.github/workflows/ci.yml

```diff
@@ -15,20 +15,14 @@ jobs:
         - centos-9
         - fedora-35
         - fedora-36
+    container:
+      image: quay.io/ovirt/imageio-test-${{matrix.distro}}
+      # Required to create loop devices.
+      options: --privileged
```

Comment:
Nice, I think this was not possible when we add the CI, or maybe we missed this option.
vdsm ci can be also be simplified using this.